pkg/manifest/ostree_deployment.go

```diff
@@ -481,3 +494,34 @@ func (p *OSTreeDeployment) getInline() []string {
 
 	return inlineData
 }
+
+// Creates systemd unit stage by ingesting the servicename and mount-points
+func createMountpointService(serviceName string, mountpoints []string) *osbuild.SystemdUnitCreateStageOptions {
+	var conditionPathExists []string
+	for _, mountpoint := range mountpoints {
+		conditionPathExists = append(conditionPathExists, "|!"+mountpoint)
+	}
+	unit := osbuild.Unit{
+		Description:         "Generated by osbuild/images - Create mountpoints if does not exist",
+		ConditionPathExists: conditionPathExists,
+	}
+	service := osbuild.Service{
+		Type:            osbuild.Oneshot,
+		RemainAfterExit: true,
+		ExecStartPre:    []string{"chattr -i"},
+		ExecStopPost:    []string{"chattr +i"},
```

Comment:
Shouldn't these specify a path to `chattr`?

Reply:
Right, the workarounds there are running `chattr -i /` (on root).
`chattr -i` without a path isn't a valid command.

Reply:
my bad.. overlooked `/`